src/structures/VoiceChannel.js

```diff
@@ -0,0 +1,14 @@
+const Base = require('./Base');
```

Comment:
First of all, let's organize the code so that everything runs smoothly, this semicolon is unnecessary
```suggestion
const Base = require('./Base')
```